R/compute_duration.R

```diff
@@ -212,3 +219,58 @@ compute_duration <- function(start_date,
   }
   duration
 }
+
+#' Map common units of time into standardized terms
+#'
+#' @param time_unit input unit
+#'
+#' @return standardized term if mapping is available
+#'
+#' @keywords internal
+#' @family internal
+#'
+#' @noRd
+valid_time_mappings <- function(time_unit) {
```

Comment:
According to our programming strategy function names should start with a verb. So it should be something like `get_unified_time_unit()`.